src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -128,6 +180,23 @@ public static String powers(int number) {
      */
     public static String arithmetic(int a, int b) {
 
-        return "";
+        int firstInteger = a;
+        int secondInteger = b;
+        int sum = firstInteger + secondInteger;
+        int dif = firstInteger - secondInteger;
+        int mult = firstInteger * secondInteger;
+        float med = ( (firstInteger + secondInteger) / 2);
```

Comment:
You need to convert ints to floats before the calculation, I suppose.
Dose it work?

Reply:
`float med = ((float)(firstInteger + secondInteger) / 2);`